DevOps/frankfurter/scripts/tools/sendlogs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#make sure to install zip command using sudo apt install.. 
+#added zip pkg to master_frankfurter setup
+
+HOST=hostname
+ID=192.168.7.1
+
+HOME=ubuntu
+USER=192.168.7.2
+
+#format: ssh ubuntu 192.168.7.2 "Linux commands"
+#makes a temp directory and goes into the temp directory once made
+#gets the logs from the services in the beaglebone and logs in temp directory
+#zips log files and prints the zip file name if it has been created
+#exit ssh
+
+ssh $HOME@$USER "
+mkdir temp
+cd temp
+
+journalctl -e -u update.service > update.log
+journalctl -e -u runtime.service > service.log
+journalctl -e -u networking.service > networking.log
+bzip2 * 
+
+cd 
+
+find temp -not -name *bz2 | xargs tar -jcvf archivelogs.tar.bz2 
+
+exit "
```

Comment:
Doesn't look like `temp` is deleted, so it will reside in `~` forever, which causes `mkdir` to fail if you run the script twice on the same machine (`mkdir -p` succeeds if a directory already exists).

My suggestion is to use a `/tmp/log-fetch` subdirectory when preparing the file, since Ubuntu cleans up old files in `tmp` for you.